demo/kubernetes/rook-pool.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: rook.io/v1beta1
+apiVersion: rook.io/v1alpha1
```

Comment:
Just wondering why do we have this yaml file example if this is already done in rook-storageclass.yaml